README.md

```diff
@@ -233,6 +233,8 @@ export LAUNCHER_MISSIONCONTROL_GITHUB_USERNAME=`git config github.user`
 export LAUNCHER_MISSIONCONTROL_GITHUB_TOKEN=`git config github.token`
 export LAUNCHER_MISSIONCONTROL_GITLAB_USERNAME=`git config gitlab.user`
 export LAUNCHER_MISSIONCONTROL_GITLAB_PRIVATE_TOKEN=`git config gitlab.token`
+export LAUNCHER_MISSIONCONTROL_BITBUCKET_USERNAME=BitbucketFakeUserName
+export LAUNCHER_MISSIONCONTROL_BITBUCKET_APPLICATION_PASSWORD=BitbucketFakeApplicationPassword1234$
```

Comment:
We're also missing a section on setting up a BitBucket account here: https://github.com/ia3andy/launcher-backend/blob/fc1bffdb618f32fe12220cc1bc523b041c50da07/README.md#prerequisites-to-run-integration-tests